scripts/convert_stable_diffusion_3_checkpoint_to_onnx.py

```diff
@@ -0,0 +1,292 @@
+    output_path = Path(output_path)
+
+    # TEXT ENCODER
+    num_tokens = pipeline.text_encoder.config.max_position_embeddings
+    text_hidden_size = pipeline.text_encoder.config.hidden_size
+    text_input = pipeline.tokenizer(
+        "A sample prompt",
+        padding="max_length",
+        max_length=pipeline.tokenizer.model_max_length,
+        truncation=True,
+        return_tensors="pt",
+    )
+    onnx_export(
+        pipeline.text_encoder,
+        # casting to torch.int32 until the CLIP fix is released: https://github.com/huggingface/transformers/pull/18515/files
+        model_args=(
+            text_input.input_ids.to(device=device, dtype=torch.int32),
+            None,
+            None,
+            None,
+            True,
+        ),
+        output_path=output_path / "text_encoder" / "model.onnx",
+        ordered_input_names=["input_ids"],
+        output_names=["last_hidden_state", "pooler_output", "hidden_states"],
+        dynamic_axes={
+            "input_ids": {0: "batch", 1: "sequence"},
+        },
+        opset=opset,
+    )
+    del pipeline.text_encoder
+
+    num_tokens = pipeline.text_encoder_2.config.max_position_embeddings
+    text_hidden_size = pipeline.text_encoder_2.config.hidden_size
+    text_input = pipeline.tokenizer_2(
+        "A sample prompt",
+        padding="max_length",
+        max_length=pipeline.tokenizer_2.model_max_length,
+        truncation=True,
+        return_tensors="pt",
+    )
+    onnx_export(
+        pipeline.text_encoder_2,
+        # casting to torch.int32 until the CLIP fix is released: https://github.com/huggingface/transformers/pull/18515/files
+        model_args=(
+            text_input.input_ids.to(device=device, dtype=torch.int32),
+            None,
+            None,
+            None,
+            True,
+        ),
+        output_path=output_path / "text_encoder_2" / "model.onnx",
+        ordered_input_names=["input_ids"],
+        output_names=["last_hidden_state", "pooler_output", "hidden_states"],
+        dynamic_axes={
+            "input_ids": {0: "batch", 1: "sequence"},
+        },
+        opset=opset,
+    )
+    del pipeline.text_encoder_2
+
+    text_input = pipeline.tokenizer_3(
+        "A sample prompt",
+        padding="max_length",
+        max_length=pipeline.tokenizer_3.model_max_length,
+        truncation=True,
+        return_tensors="pt",
+    )
+    onnx_export(
+        pipeline.text_encoder_3,
+        # casting to torch.int32 until the CLIP fix is released: https://github.com/huggingface/transformers/pull/18515/files
+        model_args=(text_input.input_ids.to(device=device, dtype=torch.int32)),
+        output_path=output_path / "text_encoder_3" / "model.onnx",
+        ordered_input_names=["input_ids"],
+        output_names=["last_hidden_state"],
+        dynamic_axes={
+            "input_ids": {0: "batch", 1: "sequence"},
+        },
+        opset=opset,
+    )
```

Comment:
Can you also collate the external tensor files into one file for this model? Like what you are doing with the transformer.